---
title: "UCP health checks fail, unable to find user nobody: no matching entries in passwd file observed"
id: 2175
draftstate: inactive
deleted: false
source: https://success.docker.com/@api/deki/pages/2175/contents
tags:
- tag: "article:errors"
- tag: "component:ucp"
- tag: "expires:nextrelease"
- tag: "platform:linux"
- tag: "product:datacenter"
- tag: "stage:reviewed"
- tag: "testedon:ucp-2.1.4"
- tag: "usecase:caas"
---
{% raw %}
<div class="mt-section" id="section_1" mt-section-origin='KBase/UCP_health_checks_fail,_"unable_to_find_user_nobody:_no_matching_entries_in_passwd_file"_observed'>
 <span id="Issue">
 </span>
 <h2 id="1-0">
  Issue
 </h2>
 <ul>
  <li>
   Suddenly UCP health checks begin to fail on nodes, and a
   <code>
    docker inspect
   </code>
   of nodes which appear to show an 'unhealthy' status in
   <code>
    docker ps
   </code>
   output reports an exit status of:
  </li>
 </ul>
 <pre>
<code>            "Health": {
                "Status": "unhealthy",
                "FailingStreak": 812,
                "Log": [
                    {
                        "Start": "2017-07-05T15:50:57.765421798-04:00",
                        "End": "2017-07-05T15:50:57.766049318-04:00",
                        "ExitCode": -1,
                        "Output": "unable to find user nobody: no matching entries in passwd file"
                    },
</code></pre>
 <ul>
  <li>
   When attempting to run
   <code>
    docker exec
   </code>
   commands, the following error is observed:
  </li>
 </ul>
 <pre>
<code>unable to find user nobody: no matching entries in passwd file
</code></pre>
 <ul>
  <li>
   The UCP cluster appears to otherwise be healthy, even though health checks seemingly fail.
  </li>
 </ul>
</div>
<div class="mt-section" id="section_2" mt-section-origin='KBase/UCP_health_checks_fail,_"unable_to_find_user_nobody:_no_matching_entries_in_passwd_file"_observed'>
 <span id="Prerequisites">
 </span>
 <h1 id="0-0">
  Prerequisites
 </h1>
 <p>
  This issue has been observed on the following versions:
 </p>
 <ul>
  <li>
   Universal Control Plane: 2.1.4
  </li>
  <li>
   Engine: 17.03.1-ee
  </li>
 </ul>
</div>
<div class="mt-section" id="section_3" mt-section-origin='KBase/UCP_health_checks_fail,_"unable_to_find_user_nobody:_no_matching_entries_in_passwd_file"_observed'>
 <span id="Steps">
 </span>
 <h1 id="0-1">
  Steps
 </h1>
 <p>
  Currently there is no resolution to this issue.Â It is being tracked via an internal bug.
 </p>
 <div class="mt-section" id="section_4" mt-section-origin='KBase/UCP_health_checks_fail,_"unable_to_find_user_nobody:_no_matching_entries_in_passwd_file"_observed'>
  <span id="Workaround">
  </span>
  <h2 id="1-1">
   Workaround
  </h2>
  <p>
   Restart the Docker Engine to temporarily resolve this issue:
  </p>
  <pre>
<code>$ sudo systemctl restart docker
</code></pre>
  <p>
   The health checks should begin to succeed again, and the cluster should report a healthy status.
  </p>
 </div>
</div>
{% endraw %}
